.github/workflows/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,37 @@
+name: 🚀 Feature request
+description: Add contributors to guestbook 💡
+title: "Feature:"
+labels: [💡 feature]
+body:
+  - type: textarea
+    attributes:
+      label: Description
+      description: Please describe the issue — for example, "As part of the Intro to Open Source course, we are encouraged to contribute to this repo.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Suggested solution
+      description: Suggest a solution — for example, "Add myself as a contributor."
+  - type: input
+    id: context
+    attributes:
+      label: Additional context
+      description: Add any other context about the problem or helpful links here.
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      description: By submitting this issue, you agree to follow our [Code of Conduct](https://docs.opensauced.pizza/contributing/code-of-conduct/)
+      options:
+        - label: I agree to follow this project's Code of Conduct
+          required: true
+  - type: checkboxes
+    id: contribution
+    attributes:
+      label: Getting Started Instructions
+      description: Please read [Getting Started](https://intro.opensauced.pizza/#/05-how-to-contribute-to-open-source?id=getting-started) to add yourself as a contributor.
+      options:
+        - label: I agree to follow this project's Getting Started instructions
+          required: true
+      description: Please read [Getting Started](https://intro.opensauced.pizza/#/05-how-to-contribute-to-open-source?id=getting-started) to add yourself as a contributor.
```

Comment:
I'm not sure why this description is repeated.

Reply:
> I'm not sure why this description is repeated. 

Oops that was a mistake @BekahHW

Reply:
Made the change @BekahHW